### DISCUSSION TITLE
How can I start with veramo ?

### DISCUSSION THREAD

**Discussion by MaryemHadjWannes** [decentralized-identity/veramo#1153] [Q&A]
How can I start with veramo ?
I want to create DIDs with ethereum addresses, create Credentials,  and verify it ? any help please !

**Comment by nickreynolds** [comment]
There are a few different ways you can get started depending on your goals/platform.

https://veramo.io/docs/node_tutorials/node_setup_identifiers

^ This is a tutorial for how to create new `did:ethr` identifiers and use them to create and verify credentials within a node.js application.

If you want to use an *existing* ethereum address (such as one managed by metamask) it's slightly more complicated, but still very doable. If you want to use metamask+browser to achieve this, let me know, and I will try to find more specific documentation.

**Reply by MaryemHadjWannes** [reply]
Thank you for your response. I wanted to further clarify the main use of Veramo in my project. I am working on a project that involves Ethereum blockchain, smart contracts and ReactJS. The goal of the project is to provide the actors with DID (decentralized identifiers) on the Ethereum blockchain and issue credentials from issuer to holder and also that can be verified by a third party.
